css/two_column.css

```diff
@@ -0,0 +1,120 @@
+html { 
+    font-size: 15px; 
+} 
+
+@media (min-width: 1500px) { 
```

Comment:
I think this is too big. I'd like to stick to common bootstrap dimensions, if possible. They've been tested with a variety of screen dimensions, and the column dimensions work well with the defaults (using bootstrap's grid defaults).

Check out http://getbootstrap.com/css/#grid for what the defaults are, and how to select the different breakpoints. There is a large devices that starts at 1170px.

Traditionally, we've considered the case of the laptop with smaller resolutions, such as breaking at 970px.